pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/StudyGovernanceDao.sql.stg

```diff
@@ -62,6 +64,7 @@ select auc.age_up_candidate_id,
   from age_up_candidate as auc
   join umbrella_study as study on study.umbrella_study_id = auc.study_id
   join user as participant on participant.user_id = auc.participant_user_id
+  join user as operator on operator.user_id = auc.operator_user_id
```

Comment:
since its inner join we need to ensure we backfill old data with operator_user_id

Reply:
I put the query which sets the operator from governance if found or participant_user_id otherwise

Reply:
On dev it fills 100% of candidates (550) with correct operator_user_id.